### PR TITLE
Add and enable enableEncryptionFormatAll

### PR DESCRIPTION
**Pull request by vermashi**

### CODE REVIEW

File: VMEncryption/main/handle.py [hunk 1]
```diff
@@ -1261,15 +1261,60 @@ def decrypt_inplace_with_separate_header_file(passphrase_file,
...
+                           status_code=str(CommonVariables.success),
+                           message=msg)
+
+    return encrypt_format_device_items(passphrase_file, device_items_to_encrypt, disk_util, True)
+
+def encrypt_format_device_items(passphrase, device_items, disk_util, force=False):
+    """
+    Formats the block devices represented by the supplied device_item.
+
+    This is done by contstructing a disk format query based on the supplied device items
```
Comment (ejarvi) [inline VMEncryption/main/handle.py]: typo ... "constructing"

File: VMEncryption/main/handle.py [hunk 2]
```diff
@@ -1278,13 +1323,19 @@ def enable_encryption_all_in_place(passphrase_file, encryption_marker, disk_util
...
            not any(di.name == device_item.name for di in device_items_to_encrypt):
             device_items_to_encrypt.append(device_item)
+    return device_items_to_encrypt
+
+
+def enable_encryption_all_in_place(passphrase_file, encryption_marker, disk_util, bek_util):
+    """
+    if return None for the success case, or return the device item which failed.
+    """
+    logger.log(msg="executing the enableencryption_all_inplace command.")
```
Comment (ejarvi) [inline VMEncryption/main/handle.py]: underscores don't match ... "enable_encryption_all_in_place"

File: VMEncryption/main/handle.py [hunk 3]
```diff
@@ -1261,15 +1261,60 @@ def decrypt_inplace_with_separate_header_file(passphrase_file,
...
+    logger.log(msg="executing the enable_encryption_all_format command")
+
+    device_items = find_all_devices_to_encrypt(encryption_marker, disk_util, bek_util)
+    # Don't encrypt partitions that are not even mounted
+    device_items_to_encrypt = filter(lambda di: di.mount_point is not None and di.mount_point != "", device_items)
+
+    msg = 'Encrypting and formatting {0} data volumes'.format(len(device_items_to_encrypt))
+    logger.log(msg);
+
+    hutil.do_status_report(operation='EnableEncryptionAllFormat',
```
Comment (ejarvi) [inline VMEncryption/main/handle.py]: Inconsistent naming - other places in code refer to 'EnableEncryptionFormatAll' where "All" is the suffix

**Comment by ejarvi** [comment]
@vermashi please update with status, this change has not yet been merged and seems to include reboot code similar to the work that Vishal is doing in separate PR

**Comment by vermashi** [comment]
Already merged.

**Pull request closed without merging**